src/get-external-builtins.js

```diff
@@ -0,0 +1,96 @@
+/********************************************************************************
+ * Copyright (C) 2022 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+// @ts-check
+
+/**
+ * Clone and check-out the vscode external builtin extensions repositories, each to the version
+ * that's expected for the current vscode baseline (as defined in vscode's product.json)
+ */
+const { run, externalBuiltinsRepos, vscode } = require('./paths.js');
+const fs = require('fs-extra');
+
+CloneCheckoutExternalBuiltins();
+
+async function CloneCheckoutExternalBuiltins() {
+    /** Path of `vscode` product.json file */
+    const prod = vscode('product.json');
+    const content = fs.readFileSync(prod, 'utf-8');
+    /** 
+     * vscode product.json section where we find info about external builtins 
+     * @type ProductBuiltInExtensionEntry[]
+    */
+    const prodJsonExts = JSON.parse(content).builtInExtensions;
+
```

Comment:
shouldn't we cleanup the `externalBuiltinsRepos` directory before entering the loop?
otherwise, we may end up preserving an old version of the plugin

Reply:
The presence of old runs should not interfere with the script, even if we switch `vscode` baseline along the way. 

That being said, it's generally a good thing to start from a known, clean state. So, I have added a `yarn clean` instruction (that calls `git clean -ffdx`) towards the top of the `README-3PP-LICENNSE-CHECK_AND_AUDIT.md`, that will clear the external builtin repos among other things.